lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -307,6 +307,42 @@ namespace Js
 
     void WasmLibrary::WasmLoadIndirectFunctionTables(Wasm::WasmModule * wasmModule, ScriptContext* ctx, Var** indirectFunctionTables, Var* localModuleFunctions, Var* importFunctions)
     {
+        ArenaAllocator alloc(_u("WasmTempIndirectTable"), ctx->GetThreadContext()->GetPageAllocator(), Js::Throw::OutOfMemory);
```

Comment:
I wonder if we should coalesce equivalent signatures earlier on (i.e. before we even emit the indirect calls)?